tests/test_renderer.py

```diff
@@ -95,7 +95,8 @@ def test_add_legend_loc(loc):
 
 def test_add_legend_no_face(sphere):
     pl = pyvista.Plotter()
-    pl.add_mesh(sphere, label='sphere')
+    sphere.point_data["Z"] = sphere.points[:, 2]
+    pl.add_mesh(sphere, scalars='Z', label='sphere')
```

Comment:
Sorry, I don't see why this was failing otherwise. Just making sure there's no unrelated bug we're hiding here.